src/h5cpp/datatype/enum.cpp

```diff
@@ -91,6 +92,24 @@ std::string Enum::name(size_t index) const
   return name;
 }
 
+bool Enum::is_bool() const {
```

Comment:
I would not have this in the Enum class. The names being "FALSE" and "TRUE" are part of the definition of EBool and Enum has no business knowing this. Also, I am concerned that you need to actually retrieve the strings in order to determine this. It seems like a rather heavy operation for answering such a simple question. Are there any alternativeS?

Reply:
Hi @martukas. Yes, you are right. It can be implemented on client side especially its implementation is not clear (or as a helper function). 
I do not have any alternative how to do it because `EBool` is really  `Enum` so you can found it only on its content.

So for now it will remove it.